.github/workflows/issue-view.yml

```diff
@@ -0,0 +1,405 @@
+                    results = await github.search.issuesAndPullRequests(
+                    { q: `repo:${repo} is:issue is:open` }
+                    )
+                  }
+                  else if(label === "closed"){
+                    results = await github.search.issuesAndPullRequests(
+                    { q: `repo:${repo} is:issue is:closed closed:>${date("-30 days")}`}
+                    )
+                  }
+                  else if(label === "stale") {
+                    results = await github.search.issuesAndPullRequests(
+                    { q: `repo:${repo} is:issue is:open created:<${date("-90 days")}`}
+                    )
+                  }
+                  else{
+                    results = await github.search.issuesAndPullRequests(
+                    { q: `repo:${repo} is:issue is:open label:${label}`}
+                    )
+                  }
+                  if(results.data.items.length === 0){
+                    return '0'
+                  }
+                  else{
+                    return results.data.total_count 
+                  }
+                }
+                userdata.url = async function(repo,label) {
+                  sleep(2000)
+                  var url;
+                  var today = new Date();
+                  var pastDate = new Date(today);
+                  if(label === "" ) {
+                    url = 'https://github.com/'+repo+'/issues?q=is%3Aissue+is%3A'+ 'open'
+                  }
+                  else if(label === "closed"){
+                    pastDate.setDate(pastDate.getDate() - 30);
+                    var closedDate = pastDate.toISOString().split("T")[0];
+                    url = 'https://github.com/' + repo + '/issues?q=is%3Aissue+is%3A' + 'closed' + '+closed%3A>'+ closedDate
+                  }
+                  else if(label === "stale") {
+                    pastDate.setDate(pastDate.getDate() - 90);
+                    var staledate = pastDate.toISOString().split("T")[0];
+                    url = 'https://github.com/' + repo + '/issues?q=is%3Aissue+is%3A' + 'open' + '+created%3A<'+staledate
+                  }
+                  else{
+                    url = 'https://github.com/'+repo+'/issues?q=is%3Aissue+is%3A'+'open'+'+label%3A'+label
+                  }
+                  return url
+                } 
+            sections:
+            - title: ''
+              widgets:
+              - type: 'table'
+                headers:
+                - 'Action'
+                - 'Open'
+                - 'Need-to-triage'
+                - 'Enhancement'
+                - 'Question'
+                - 'Bug'
+                - 'Documentation'
+                - 'Performance-issue'
+                - 'Waiting-for-customer'
+                - 'External-dependency'
+                - 'Idle(inactive>14days)'
+                - 'Stale(90_days old)'
+                - 'Closed(last_30 days)'
+                elements:
```

Comment:
We are using both "-"(_hyphen)_ and "_" _(underscore)_ . Can we just use one to have a better UI experience?

Reply:
hyphen is giving line break here for stale and closed. In the label names can't change it as they are represented with hyphen in github UI as well.

Reply:
cool. so for labels we'll have hyphen and the meanings such as 90 days old, we'll have underscore, right?

Reply:
yes